pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
Should go in its own wrapper library. This will cause conflicts with docker-plugin if we pull it in ourselves without adding the classloader masking tricks... which would impact testing. There are four routes I can see:

1. Add a temporary dependency on `docker-plugin` as that currently "provides" the API
2. Create the wrapper plugin and add the pull request to switch to the wrapper plugin to `docker-plugin`
3. Use maven-shade-plugin to move the dependency into a private package space (to prevent classloader conflicts)
4. Use the poorly documented classloader masking options in hpi to hide the dependency and prevent conflicts

I do not thing we can ignore this issue for the release